Assets/Scripts/RayMarching.cs

```diff
@@ -109,6 +109,7 @@ void OnPreRender()
         {
             m_CommandBuffer = new CommandBuffer();
             m_CommandBuffer.name = "Ray Marching";
+            m_CommandBuffer.SetGlobalFloat("_YAxisRaySign", SystemInfo.graphicsDeviceVersion.StartsWith("Direct") ? -1f : 1f);
```

Comment:
No better way of doing this? I won't allow this, shaders have macros defined that identify the platform